week-2/G-conditionals-2/exercise-4.js

```diff
@@ -9,7 +9,12 @@
 */
 
 function containsCode(sentence) {
-
+  var sentChk = sentence.match("code");
+  if (sentChk != null) {
+    return "True";
+  } else {
+    return "Flase";
+  }
```

Comment:
:+1: Good, but you should use a `boolean` isntead of a `string` for `true`/`false`.